src/hooks/useTransactionStatus.ts

```diff
@@ -1,10 +1,12 @@
+import { ReplaceTxHoverContext } from '@/components/transactions/GroupedTxListItems/ReplaceTxHoverProvider'
 import { useAppSelector } from '@/store'
 import { PendingStatus, selectPendingTxById } from '@/store/pendingTxsSlice'
 import { isSignableBy } from '@/utils/transaction-guards'
 import { TransactionSummary, TransactionStatus } from '@gnosis.pm/safe-react-gateway-sdk'
+import { useContext } from 'react'
 import useWallet from './wallets/useWallet'
 
-type TxLocalStatus = TransactionStatus | PendingStatus
+type TxLocalStatus = TransactionStatus | PendingStatus | 'WILL_BE_REPLACED'
```

Comment:
Can you please put it in a var so that we don't have to type the string again on line 36?

E.g.
```
const ReplacedStatus = 'WILL_BE_REPLACED'
...
const STATUS_LABELS = {
  [ReplacedStatus]: 'Transaction will be replaced',
}
...
return STATUS_LABELS[ReplacedStatus]
```